src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -339,13 +339,13 @@ listed below.
 
 MUST be present:                                                   
 
-| Column name  | Definition                                                                                                                                                                   |
-| ---------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| name         | REQUIRED. Name of the electrode contact point.                                                                                                                               |
-| x            | REQUIRED. X position. The positions of the center of each electrode in xyz space. Units are in millimeters or pixels and are specified in \_\*space-<label>\_electrode.json. |
-| y            | REQUIRED. Y position.                                                                                                                                                        |
-| z            | REQUIRED. Z position. If electrodes are in 2D space this should be a column of n/a values.                                                                                   |
-| size         | REQUIRED. Surface area of the electrode, in mm^2.                                                                                                                            |
+| Column name  | Definition                                                                                                                                 |
+| ---------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| name         | REQUIRED. Name of the electrode contact point.                                                                                             |
+| x            | REQUIRED. X position. The positions of the center of each electrode in xyz space. Units are specified in `*_space-<label>_electrode.json`. |
```

Comment:
```suggestion
| x            | REQUIRED. X position. The positions of the center of each electrode in xyz space. Units are specified in `*_space-<label>coordsystem.json`. |
```

I think this may be an error in the iEEG spec. AFAIK this info should be specified in COORDSYSTEM.json, ...

Reply:
agreed. Note that you have a _ missing in the corrected `*_space-<label>_coordsystem.json`